conference/covid.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+---
+
+# COVID-19 decision timeline
+
+We in the [organizing committee](/conference/committees/) are and will be very
+careful about minimizing health risk for all participants and those affected by
+participants who travel to the event.
+
+We will be closely monitoring the COVID-19 situation in Sweden and other
+countries so that we can decide whether this event can take place in person
+December 1-2, 2020.
```

Comment:
```suggestion
December 1-2, 2020. We will not just consider if we *are allowed to* host
this conference, but if it is *responsible* to host it.
```
This is my own editorial opinion which I accept if others don't agree with.  To review: does this reflect our opinions?

Reply:
Thank you. Much better wording and for me the responsibility part is number one.